### PR TITLE
ref $window.appConfig as service

### PR DESCRIPTION
**Pull request by JDonadio**
Fixes https://github.com/bitpay/copay/issues/5315

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
appConfigService needs to be add on incomingData controller

File: src/js/controllers/tab-home.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 'use strict';
 
 angular.module('copayApp.controllers').controller('tabHomeController',
-  function($rootScope, $timeout, $scope, $state, $stateParams, $ionicModal, $ionicScrollDelegate, gettextCatalog, lodash, popupService, ongoingProcess, externalLinkService, latestReleaseService, profileService, walletService, configService, $log, platformInfo, storageService, txpModalService, $window, bitpayCardService, startupService, addressbookService, feedbackService) {
+  function($rootScope, $timeout, $scope, $state, $stateParams, $ionicModal, $ionicScrollDelegate, gettextCatalog, lodash, popupService, ongoingProcess, externalLinkService, latestReleaseService, profileService, walletService, configService, $log, platformInfo, storageService, txpModalService, appConfigService, bitpayCardService, startupService, addressbookService, feedbackService) {
+    console.log(appConfigService.version);
```
Comment (gabrielbazan7) [inline src/js/controllers/tab-home.js]: remove console.log

File: src/js/controllers/tab-home.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
     var wallet;
     var listeners = [];
     var notifications = [];
     $scope.externalServices = {};
     $scope.openTxpModal = txpModalService.open;
-    $scope.version = $window.version;
-    $scope.name = $window.appConfig.nameCase;
+    $scope.version = appConfigService;
```
Comment (gabrielbazan7) [inline src/js/controllers/tab-home.js]: appConfigService.version ?

File: src/js/controllers/feedback/send.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ angular.module('copayApp.controllers').controller('sendController', function($sc
       "Email": lodash.values(config.emailFor)[0] || ' ',
       "Feedback": goHome ? ' ' : feedback,
       "Score": $stateParams.score || ' ',
-      "AppVersion": $window.version,
+      "AppVersion": appConfigService.version,
```
Comment (gabrielbazan7) [inline src/js/controllers/feedback/send.js]: This is ok ?

File: src/js/controllers/preferencesAbout.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
 'use strict';
 
 angular.module('copayApp.controllers').controller('preferencesAbout',
-  function($scope, $window, gettextCatalog, externalLinkService) {
+  function($scope, $window, appConfigService, gettextCatalog, externalLinkService) {
 
-    $scope.title = gettextCatalog.getString('About') + ' ' + $window.appConfig.nameCase;
-    $scope.version = $window.version;
+    $scope.title = gettextCatalog.getString('About') + ' ' + appConfigService.nameCase;
+    $scope.version = appConfigService.version;
```
Comment (gabrielbazan7) [inline src/js/controllers/preferencesAbout.js]: Also here $window.version == $window.appConfig.version ?

**Comment by JDonadio** [comment]
Is exactly the same, but we shouldn't change it anyways. Updated

**Pull request merged**